crates/next-core/src/next_font_google/request.rs

```diff
@@ -0,0 +1,35 @@
+use serde::Deserialize;
+
+/// The top-most structure encoded into the query param in requests to
+/// `@next/font/google` generated by the @next/font swc transform. e.g.
+/// `@next/font/google/target.css?{"path": "index.js", "import": "Inter"...`
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NextFontRequest {
+    pub path: String,
+    pub import: String,
+    pub arguments: (NextFontRequestArguments,),
```

Comment:
Great question — from what I can tell, the swc transform seems to always create things this way, a json array always of size one. I tried a few different variations on importing and always got this result. @hanneslund do you have any insight?

Reply:
`arguments` is an array of the function call arguments.
```jsx
const font = Font() // arguments: []
const font = Font(1,{},3) // arguments: [1,{},3]
```
Currently the font functions don't have more than one parameter. So for now it will only be empty or have one element.